packages/core/entity/FileDetail/RenderZarrThumbnailURL.ts

```diff
@@ -0,0 +1,101 @@
+import * as zarr from "@zarrita/core";
+import { FetchStore } from "@zarrita/storage";
+import { get as zarrGet } from "@zarrita/indexing";
+
+interface AxisData {
+    name: string;
+    type: "time" | "channel" | "space";
+    unit?: string;
+}
+
+// Define the type for the transformed array items
+interface TransformedAxes {
+    name: string;
+    value: number | null;
+}
+
+// Function to transform the array
+function transformAxes(originalArray: AxisData[]): TransformedAxes[] {
+    return originalArray.map((item) => ({
+        name: item.name,
+        value: item.type !== "space" ? 0 : null,
+    }));
+}
+
+export async function renderZarrThumbnailURL(zarrUrl: string): Promise<string> {
```

Comment:
Could you add a comment describing this function and/or more comments across the function to explain the various things going on?

Reply:
added in [bde338a](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/bde338ab3990114084fbbb8f912075627583a5c8)[bde338a](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/bde338ab3990114084fbbb8f912075627583a5c8)[bde338a](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/143/commits/bde338ab3990114084fbbb8f912075627583a5c8)